packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx

```diff
@@ -121,14 +121,14 @@ export function getStyles(props: ISearchBoxStyleProps): ISearchBoxStyles {
         flexBasis: '32px',
         flexShrink: 0,
         padding: 1,
-        color: palette.themePrimary,
-        backgroundColor: 'transparent'
+        color: palette.themePrimary
       }
     ],
     field: [
       'ms-SearchBox-field',
       normalize,
       {
+        backgroundColor: 'transparent',
```

Comment:
@phkuo do you know if 'transparent'  as opposed to the semantic slot of inputBackground is right here? FWIW, Textfield uses Transparent as a literal as well.

Reply:
We want the field itself to be transparent and the root element to use the `inputBackground` slot so the background color envelops the search icon and clear button as well.